packages/docz-tools/src/gatsby-theme-docz/theme/space.ts

```diff
@@ -0,0 +1,11 @@
+export const space = {
```

Comment:
I kind of wonder if we should make a followup to have these generated.

Reply:
I think we can look at actually getting off of theme ui as a follow up task. Might make more sense for us.